src/validation.cpp

```diff
@@ -3697,7 +3751,9 @@ bool RewindBlockIndex(const CChainParams& params)
             // of the blockchain).
             break;
         }
-        if (!DisconnectTip(state, params, true)) {
+        DisconnectedBlockTransactions dummypool;
+        // fBare=true means nothing will actually be added to dummypool
```

Comment:
Hmm, thats a super confusing API now....maybe either pass in a pointer to a DisconnectedBlockTransactions as both fBare and the pool?